broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -212,16 +413,24 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(ServiceBindingNotFound, gone);
   }
 
-  validateRequest(req, res) {
-    /* jshint unused:false */
-    if (req.instance.async && (_.get(req, 'query.accepts_incomplete', 'false') !== 'true')) {
-      throw new UnprocessableEntity('This request requires client support for asynchronous service operations.', 'AsyncRequired');
+  deleteBindingV2(req, res) {
+    const params = _(req.query)
```

Comment:
same comment as above. how diff is this method from v1 counterpart?

Reply:
Done